src/main/kotlin/no/nav/familie/klage/formkrav/FormService.kt

```diff
@@ -19,13 +21,15 @@ class FormService(
     private val formRepository: FormRepository,
     private val stegService: StegService,
     private val behandlingService: BehandlingService,
-    private val vurderingService: VurderingService
+    private val vurderingService: VurderingService,
+    private val taskRepository: TaskRepository
 ) {
 
     fun hentForm(behandlingId: UUID): Form = formRepository.findByIdOrThrow(behandlingId)
 
     @Transactional
     fun opprettInitielleFormkrav(behandlingId: UUID): Form {
+        taskRepository.save(BehandlingsstatistikkTask.opprettPåbegyntTask(behandlingId = behandlingId))
```

Comment:
Forslag til løsning;
1. Flytt denne til `oppdaterForm`
2. Hent ut behandlingshistorikk for behandlingId
3. Hvis det IKKE finnes behandlingshistorikk for `StegTyoe.FORMKRAV` så oppretter vi en task for påbegynt arbeid
4. Ellers dropper vi det

Andre tanker?

Reply:
Høres fint ut 👍